app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] If comment belongs to author then I think you'll kindly make author_id foreign key in the comments table instead of the default user_id